apps/src/aiTutor/views/legacyLabs/AiTutorContainer.tsx

```diff
@@ -22,12 +25,14 @@ const aiTutorHelper = new AiTutorLegacyLabContextHelper();
 
 interface Level {
   longInstructions?: string;
+  hideSource?: boolean;
```

Comment:
🤔 There might be some nuance with `hideSource` and `isReadOnlyWorkspace` https://github.com/code-dot-org/code-dot-org/blob/914a398fbb9eb70495e1fb89897b81f8941384cd/apps/src/redux/pageConstants.js#L24 

If `hideSource` is true then we don't see the code workspace, which totally matches the intent here. I wonder what should happen if `isReadOnlyWorkspace` is true - in that case the student can't edit the code, but they can still see it. So, presumably, the Tutor still be available and would reference the code but not tell them to make any changes? 

Beyond the scope of this change, and likely related to @dmcavoy's modes diagram. Maybe add it to the list of things to sync about?